### DISCUSSION TITLE
React native field onFocus in Submit button clicked

### DISCUSSION THREAD

**Discussion by minhaz19** [react-hook-form/react-hook-form#10079] [Q&A] [answered]
When I have a big form in react-hook-form How to redirect/scroll to the first error filed after clicking on Submit button in React Native?

**Comment by bluebill1049** [comment]
Have you tried to wrap your component with Controller, and make sure your component does offer `ref.focus`?

https://react-hook-form.com/api/usecontroller/

**Comment by leapful** [comment, marked as answer]
@minhaz19 

This question is not related to react-hook-form, it's about how to handle scroll view in react-native.

You can use library like https://www.npmjs.com/package/react-native-scroll-into-view to handle manually.
